### PR TITLE
new package: libnice

### PR DESCRIPTION
**Pull request by muellni**
An glib based [ICE](https://en.wikipedia.org/wiki/Interactive_Connectivity_Establishment) library.

### CODE REVIEW

File: src/libnice.mk [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+define $(PKG)_UPDATE
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
+        $(MXE_CONFIGURE_OPTS) \
+        PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config' \
+        GLIB_COMPILE_SCHEMAS='$(PREFIX)/$(TARGET)/bin/glib-compile-schemas' \
+        MAKE=$(MAKE)
+    $(MAKE) -C '$(1)'         -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS=
```
Comment (starius) [inline src/libnice.mk]:
```
$(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)' $(MXE_DISABLE_PROGRAMS)
$(MAKE) -C '$(BUILD_DIR)' -j 1 install $(MXE_DISABLE_PROGRAMS)
```

File: src/libnice.mk [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := https://nice.freedesktop.org/releases/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc glib
+
+define $(PKG)_UPDATE
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
```
Comment (starius) [inline src/libnice.mk]:
```
cd '$(BUILD_DIR)' && '$(SOURCE_DIR)/configure'
```

File: src/libnice.mk [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        $(MXE_CONFIGURE_OPTS) \
+        PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config' \
```
Comment (starius) [inline src/libnice.mk]: Does `PKG_CONFIG='$(TARGET)-pkg-config'` work?

File: src/libnice.mk [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+PKG             := libnice
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.1.13
+$(PKG)_CHECKSUM := 61112d9f3be933a827c8365f20551563953af6718057928f51f487bfe88419e1
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := https://nice.freedesktop.org/releases/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc glib
+
+define $(PKG)_UPDATE
```
Comment (starius) [inline src/libnice.mk]: Please provide an updater or
```
    echo 'TODO: write update script for $(PKG).' >&2;
    echo $($(PKG)_VERSION)
```

File: src/libnice.mk [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# This file is part of MXE.
+# See index.html for further information.
```
Comment (starius) [inline src/libnice.mk]: Replace these two lines with
```
This file is part of MXE. See LICENSE.md for licensing information.
```

File: src/libnice.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
+        $(MXE_CONFIGURE_OPTS) \
+        PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config' \
+        GLIB_COMPILE_SCHEMAS='$(PREFIX)/$(TARGET)/bin/glib-compile-schemas' \
+        MAKE=$(MAKE)
+    $(MAKE) -C '$(1)'         -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS=
+    $(TARGET)-gcc \
+        '$(1)/examples/simple-example.c' -o '$(PREFIX)/$(TARGET)/bin/test-$(PKG).exe' \
```
Comment (starius) [inline src/libnice.mk]: `'$(SOURCE_DIR)/examples/simple-example.c'`

**Comment by muellni** [comment]
Here you go... Thanks for reviewing.

**Comment by starius** [comment]
Thanks! Can you replace tabs with spaces, please?

**Comment by starius** [comment]
Tested on all 4 targets. The [list of files](https://gist.github.com/b8b48c670a69c4065fcc89b6d4561eb7) looks good to me.

**Comment by muellni** [comment]
tabs -> spaces: done

**Comment by starius** [comment]
Thanks!

**Pull request merged**